Competition2017/src/competition/subsystems/collector/CollectorSubsystem.java

```diff
@@ -35,6 +38,20 @@ public CollectorSubsystem(WPIFactory factory, XPropertyManager propManager) {
         
         collectorMotor.createTelemetryProperties("Collector motor", propManager);
         
+        lowPowerCollectorProperty = propManager.createPersistentProperty("Low power collector property", 0.3);
```

Comment:
don't include the word "property" here, since nearly everything on the dashboard, by definition, is a property =]